muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/actions/base/contacts.ts

```diff
@@ -0,0 +1,108 @@
+import { AnyActionType, SpecificActionType } from "~/actions";
+import { StateType } from "~/reducers";
+import {
+  ContactGroup,
+  ContactGroupNames,
+  Contact,
+} from "~/reducers/base/contacts";
+import { LoadingState } from "~/@types/shared";
+import notificationActions from "~/actions/base/notifications";
+import promisify from "~/util/promisify";
+import mApi, { MApiError } from "~/lib/mApi";
+import { Dispatch } from "react-redux";
+
+export type LOAD_CONTACT_GROUP = SpecificActionType<
+  "LOAD_CONTACT_GROUP",
+  ContactGroupPayload
+>;
+export type UPDATE_CONTACT_GROUP_STATE = SpecificActionType<
+  "UPDATE_CONTACT_GROUP_STATE",
+  LoadingStatePayload
+>;
+
```

Comment:
Consider renaming action types differently.

CONTACT_LOAD_GROUP
CONTACT_UPDATE_GROUP_STATE




While developing its easier to recognise action if first part tells what it is related to. In this case CONTACT. Second part tells what is happening.